shell/platform/darwin/ios/framework/Source/FlutterEngineGroup.mm

```diff
@@ -60,7 +73,13 @@ - (FlutterEngine*)makeEngineWithEntrypoint:(nullable NSString*)entrypoint
                  name:FlutterEngineWillDealloc
                object:engine];
 
-  return [engine autorelease];
+  return engine;
+}
+
+- (FlutterEngine*)createEngine {
```

Comment:
"create" has special semantics on objc.  This should be "makeEngine".

Reply:
Done